apps/modernization-ui/src/apps/page-builder/pages/BusinessRulesLibrary/RequiredFieldMarker.tsx

```diff
@@ -0,0 +1,31 @@
+import './RequiredFieldMarker.scss';
+
+interface Props {
+    functionName: string;
+    fieldName: string;
+}
+
+type RequiredFields = {
+    [key: string]: string[];
+};
+
+const RequiredFieldMarker = ({ functionName, fieldName }: Props) => {
```

Comment:
Issue: i do not believe this component is necessary. Most `input` components already provide the required indicator.